src/test/java/SplunkDockerImageTest.java

```diff
@@ -28,33 +33,7 @@ private void connectToSplunk(DockerImage dockerImage) {
 
         System.setProperty("https.protocols", "TLSv1,TLSv1.1,TLSv1.2");
 
-        // Create a Service instance and log in with the argument map
-        Service service = Service.connect(loginArgs);
-
-        System.out.println(service.getApplications());
```

Comment:
why was this removed?

Reply:
If you're asking about the sysout statement, just cleaned it as we are touching this file.
And since we aren't printing the applications, we don't need the service variable anymore, just a call to Service::connect() would suffice - https://github.com/padaiyal/jAdya/pull/24/files/2aa53544dfa59d646ce88178e23ce265a682f3d6#diff-de09bfdbf962b08e15f78e58d544e74f9dbcc9b40a4232f6426a3005450d6907R37

Reply:
Got it. So connect() would suffice in testing out the container is fully up and running?
I wanted to make it uniform, since we do some operations in other containers to test. Let me know what you think.

Reply:
Post out discussion, we decided to keep it as is i.e. connect() would suffice to test if the container is up and running.